acpreprocessing/imagej/imagej.py

```diff
@@ -0,0 +1,74 @@
+import copy
+from multiprocessing.sharedctypes import Value
+import os
+import subprocess
+
+class ImageJRunner:
+    def __init__(self,
+                 imagej_path=None,
+                 config={},
+                 script_type='macro',
+                 script_path=None,
+                 args={}):
+        if imagej_path is None:
+            self.imagej_path = os.getenv(
+                'IMAGEJ_PATH',
+                default='Fiji.app/ImageJ-linux64'
+            )
+        else:
+            self.imagej_path = imagej_path
+        
+        if 'script_type' in config:
+            self.script_type = config['script_type']
+        else:
+            self.script_type = script_type
+        
+        if 'script_path' in config:
+            self.script_path = config['script_path']
+        else:
+            self.script_path = script_path
+        
+        if 'args' in config:
+            self.args = copy.deepcopy(config['args'])
+        else:
+            self.args = copy.deepcopy(args)
+        
+        self.argstring = self._generate_argstring()
+        self.imagej_cmd = self._generate_cmd()
+        
+        if os.path.isfile(self.imagej_path):
```

Comment:
```suggestion
        if not os.path.isfile(self.imagej_path):
```
This is raiising an exception if it is a file -- we'd mostly accept it to be an executable file, right?

Reply:
Yes. This is supposed to be the path to an imagej executable